src/components/common/Footer/Footer.component.tsx

```diff
@@ -6,70 +6,77 @@ import Facebook32 from '@/assets/svg/facebook-32.svg';
 import Instagram32 from '@/assets/svg/instagram-32.svg';
 import { useRouter } from 'next/router';
 import { HOME_PAGE } from '@/constants';
+import { getRecruitingProgressStatusFromRecruitingPeriod } from '@/utils/date';
 import * as Styled from './Footer.styled';
 
 const Footer = () => {
   const { pathname: currentPage } = useRouter();
+  const recruitingProgressStatus = getRecruitingProgressStatusFromRecruitingPeriod(new Date());
   return (
-    <Styled.Footer currentPage={currentPage}>
-      <Styled.FooterInner>
-        <Styled.Copyright currentPage={currentPage}>
-          © Mash-Up 2022. Made in Seoul.
-        </Styled.Copyright>
-        <Styled.ExternalLinkWrapper currentPage={currentPage}>
-          <a
-            href="https://github.com/mash-up-kr/"
-            target="_blank"
-            rel="noreferrer"
-            aria-label="mash up github"
-          >
-            {currentPage === HOME_PAGE ? (
-              <Styled.GithubIconWrapper>
-                <Styled.GithubIconBackground />
-                <Styled.GithubIconDark aria-hidden />
-              </Styled.GithubIconWrapper>
-            ) : (
-              <GithubWhite32 aria-hidden />
-            )}
-          </a>
-          <a href="mailto:recruit.mashup@gmail.com" aria-label="mash up email">
-            <Mail32 aria-hidden />
-          </a>
-          <a
-            href="https://mash-up.tistory.com/"
-            target="_blank"
-            rel="noreferrer"
-            aria-label="mash up tistory"
-          >
-            <Tistory32 aria-hidden />
-          </a>
-          <a
-            href="https://www.behance.net/Mash-Up/"
-            target="_blank"
-            rel="noreferrer"
-            aria-label="mash up behance"
-          >
-            <Behance32 aria-hidden />
-          </a>
-          <a
-            href="https://www.facebook.com/mashupgroup/"
-            target="_blank"
-            rel="noreferrer"
-            aria-label="mash up facebook"
-          >
-            <Facebook32 aria-hidden />
-          </a>
-          <a
-            href="https://www.instagram.com/official_mashup_/"
-            target="_blank"
-            rel="noreferrer"
-            aria-label="mash up instagram"
-          >
-            <Instagram32 aria-hidden />
-          </a>
-        </Styled.ExternalLinkWrapper>
-      </Styled.FooterInner>
-    </Styled.Footer>
+    <>
+      {recruitingProgressStatus === 'PREVIOUS' && null}
+      {recruitingProgressStatus !== 'PREVIOUS' && (
+        <Styled.Footer currentPage={currentPage}>
+          <Styled.FooterInner>
+            <Styled.Copyright currentPage={currentPage}>
+              © Mash-Up 2022. Made in Seoul.
```

Comment:
```suggestion
              © Mash-Up {currentYear}. Made in Seoul.
```
해당 PR과 관련 없긴 한데 지나가다 보여서 제안 드려요~

Reply:
오~ 좋은 방법이네요